calicoctl.py

```diff
@@ -237,9 +248,13 @@ def node(ip, node_image):
     if not ipv4_pools:
         print "No IPv4 range defined.  Exiting."
         return
+    ipv6_pools = client.get_ip_pools("v6")
+    if not ipv6_pools:
```

Comment:
Is it an error to have no ipv6_pools if we're not configured with an IPv6 address? Looks like we bail in that case right now.